cost.go

```diff
@@ -0,0 +1,178 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package celvet
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/google/cel-go/cel"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation"
+	structuralschema "k8s.io/apiextensions-apiserver/pkg/apiserver/schema"
+	schemacel "k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel"
+)
+
+type costError struct {
```

Comment:
Do we need all those info in costError? What program in costError used for?

Reply:
I was eventually wanting `costError.Program` to be parsed by another function to provide cost visualization, but I don't think it does anything at the moment, so I've removed it.